### PR TITLE
fix: add asset tags to resources so they load properly in non-root directories

### LINKED ISSUE #139: Want To Install In Sub Directory 📁
### I Want to install Monica in SubDirectory 

Hi,
I Want to install Monica in SubDirectory ie. https://localhost/monica/ instead of https://localhost/
i tried but getting 404 error of assets

i am trying to install public folder in sub directory of web root

- private
  - CHANGELOG
  - CONTRIBUTING.md
  - LICENSE
  - Procfile
  - app
  - artisan
  - bootstrap
  - bower.json
  - composer.json
  - composer.lock
  - config
  - database 
  - .........
- htdocs
  - subdir
    - build
    - css
    - favicon.ico
    - fonts
    - img
    - index.php
    - js
    - robots.txt
    - web.config
    - storage

```please mention version also it helps to keep up to date```

### PR DESCRIPTION
**Pull request by bendauphinee**
I plan to make an entry once I've got more of this fixed. I wanted to get this small PR in first to make sure I've got it right about contributing.

(Partially) Fixes #139

### CODE REVIEW

File: resources/views/marketing/auth.blade.php [hunk 1]
```diff
@@ -6,10 +6,10 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <meta name="csrf-token" content="{{ csrf_token() }}"/>
     <title>@yield('title', trans('app.application_title'))</title>
-    <link rel="manifest" href="/manifest.webmanifest">
+    <link rel="manifest" href="href="{{ asset('/manifest.webmanifest') }}"">
```
Comment (asbiin) [inline resources/views/marketing/auth.blade.php]: I think it should be
```suggestion
    <link rel="manifest" href="{{ asset('/manifest.webmanifest') }}">
```

**Comment by bendauphinee** [comment]
@asbiin I flagged the review as resolved, but it's still showing as changes requested and I'm not sure how to resolve that, or if I need to. This is updated and should be clean now.

**Comment by bendauphinee** [comment]
@djaiss Is it usual for Jenkins to be broken? I take it it's broken because it's been hours and not completed.

**Comment by asbiin** [comment]
@bendauphinee Thank you!

**Comment by bkanuka** [comment]
Some resources are still missing asset tags. For example, [line 12 in verify.blade.php](https://github.com/monicahq/monica/blob/master/resources/views/auth/verify.blade.php#L12)

I'll attempt an MR but I've never used php/blade.

**Comment by bendauphinee** [comment]
@bkanuka Yes, a lot of pages are still missing asset tags. I have most of the modifications, just been busy with holiday stuff. Feel free to PR them if you want to.

**Pull request merged**